### ISSUE TITLE
[HOLD for payment 2024-01-24] [$500] Implement new layout in About page

### ISSUE THREAD

**Issue by roryabraham** [Expensify/App#26779]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

HOLD on https://github.com/Expensify/App/pull/26784

## Action Performed:
Open the `Settings` -> `About` page.

## Expected Result:
<img width="167" alt="image" src="https://github.com/Expensify/App/assets/47436092/9b9ba4d2-d725-4aa3-a150-50ac57d71aef">

Except it should also include the app version in the same font color as the `<` back button beneath the coin on the `About` page.

Notes:

- It should be implemented with the `IllustratedHeaderPageLayout`
- because the app version needs to be included in the top section, the `IllustratedHeaderPageLayout` component may need to be updated to allow us to render some stuff beneath the animation
- Currently the aspect ratio for all these animations are the same, so by setting the animation to 100% width the top section of the screen always has the same height. We need to render the app version in the top section without adjusting the animation size or the height of the top (colored) section.
- The animation lottie asset is here:

    [Loading.json.zip](https://github.com/Expensify/App/files/12524436/Loading.json.zip)
    
- The correct background color is `yellow600 ` or `#D18000`

## Actual Result:
It does not look like that.

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0179e9954261696fde</li>
        <li>Upwork Job ID: 1699055159124353024</li>
        <li>Last Price Increase: 2023-09-05</li>
<li>Automatic offers: </li>
<ul>
<li>s77rt | Reviewer | 26508154</li>
<li>Pujan92 | Contributor | 26508156</li>
</ul></li>
    </ul>
</details>

**Comment by roryabraham** [comment]
Ok, I know there's a lot of time pressure to try to write these proposals, but please try to take the time it needs to write a quality proposal. There's an additional consideration for this layout that wasn't needed for https://github.com/Expensify/App/issues/26775 (i.e: the app version text, not shown in the mockup)

**Comment by Pujan92** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Implement new layout for about page

### What is the root cause of that problem?
New feature

### What changes do you think we should make in order to solve the problem?
Add `IllustratedHeaderPageLayout` as the wrapper component and pass the current content inside it. We need to remove logo and version part from there bcoz that needs to be displayed on the animation. For showing the version number we can pass `overlayContent` function which is going to be added in this [PR](https://github.com/Expensify/App/pull/26687). With that we need to provide some top margin to align the version text below coin. Set the prop values respective to the about page(considering the right animation file and bg color defined in the themecolors).

```
       <IllustratedHeaderPageLayout
            title={translate('initialSettingsPage.about')}
            onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS)}
            illustration={LottieAnimations....}
            backgroundColor={'#D18000'}
        >
            
        </IllustratedHeaderPageLayout>
```

**Comment by DylanDylann** [comment]
@roryabraham I just updated the proposal with detail change before the later proposal.

**Comment by roryabraham** [comment]
@DylanDylann Your proposal still does not contain any information about the app version text.

**Comment by DylanDylann** [comment]
@roryabraham I just [updated proposal](https://github.com/Expensify/App/issues/26779#issuecomment-1706687097) 7 minutes ago after see your reminder

**Comment by roryabraham** [comment]
@DylanDylann as I just wrote, your proposal in its current state not sufficient or equivalent to the accepted proposal.

**Comment by Pujan92** [comment]
@roryabraham Can we put this issue on Hold until the other [PR](https://github.com/Expensify/App/pull/26687) gets merged

**Comment by roryabraham** [comment]
Sure

**Comment by roryabraham** [comment]
@Pujan92 took this off HOLD

**Comment by roryabraham** [comment]
Answered your questions there. Thanks

**Comment by roryabraham** [comment]
Posted a CTA asking for help to fix the animation file not working with the SVG renderer

**Comment by isabelastisser** [comment]
@roryabraham are you still working on this?

**Comment by roryabraham** [comment]
Hi, progress here has stalled as we are very stuck in general due to problems with the spinning coin animation.

**Comment by isabelastisser** [comment]
@roryabraham if this is not being prioritized, should we close this and reopen it when the `problems with the spinning coin animation` resolve?

**Comment by isabelastisser** [comment]
Bump @roryabraham on my question above. Thanks!

**Comment by roryabraham** [comment]
I'm not sure. Maybe we can choose another animation for this page since there's some as-yet-unidentified problem with the spinning coin animation.

cc @Expensify/design

**Comment by shawnborton** [comment]
Can you summarize again what the problem is with the existing coin animation? I can take that to Augenblick to try to get some revisions made.

**Comment by Pujan92** [comment]
@shawnborton existing coin animation isn't working properly for safari browser, may be that team can check once in any lottie preview online within safari to figure out the issue. Sharing the screencast about how it looks for me in safari.

https://github.com/Expensify/App/assets/14358475/e8adcca5-d006-4561-9bd7-68eb2f1a6262

**Comment by shawnborton** [comment]
Cool, asked internally for a new version of this. Will keep you posted!

**Comment by shawnborton** [comment]
Can you try this one? 

[Expensify-Loading-121923.json.zip](https://github.com/Expensify/App/files/13717552/Expensify-Loading-121923.json.zip)

**Comment by Pujan92** [comment]
It still isn't the smooth and proper for safari.

1) Safari

https://github.com/Expensify/App/assets/14358475/c917be16-b887-4081-9d29-a1ef83fcce93

**After converting to the .lottie file**

https://github.com/Expensify/App/assets/14358475/1e92b9e1-afc7-4947-9f7e-b0b43e78c2f2

2) Chrome

https://github.com/Expensify/App/assets/14358475/14d82c65-14c4-45e3-8db8-d28302787283

**Comment by shawnborton** [comment]
We're looking into it, will report back.

**Comment by shawnborton** [comment]
Here is a new version to try: [EXP-Loading-Coin.zip](https://github.com/Expensify/App/files/13812517/EXP-Loading-Coin.zip)

**Comment by roryabraham** [comment]
PR merged! 🚀  Thanks @shawnborton for working with our design partners to get an animation that worked as expected and was correctly sized

**Comment by s77rt** [comment]
I don't think we need a regression test here for the simplicity. This kinda of design change is not something that would just break. This is an improvement more than a new feature.

**Comment by isabelastisser** [comment]
all set!
